prem_utils/models.json

```diff
@@ -560,52 +560,62 @@
                 {
                     "slug": "phi-1-5",
                     "model_type": "text2text",
-                    "context_window": 2048
+                    "context_window": 2048,
+                    "coming_soon": true
                 },
                 {
                     "slug": "phi-2",
                     "model_type": "text2text",
-                    "context_window": 2048
+                    "context_window": 2048,
+                    "coming_soon": true
                 },
                 {
                     "slug": "tinyllama",
                     "model_type": "text2text",
-                    "context_window": 2048
+                    "context_window": 2048,
+                    "coming_soon": true
                 },
                 {
                     "slug": "mamba-chat",
                     "model_type": "text2text",
-                    "context_window": 128
+                    "context_window": 128,
+                    "coming_soon": true
                 },
                 {
                     "slug": "phi1-5-modal",
                     "model_type": "text2text",
-                    "context_window": 2048
+                    "context_window": 2048,
+                    "coming_soon": true
                 },
                 {
                     "slug": "phi2-modal",
                     "model_type": "text2text",
-                    "context_window": 2048
+                    "context_window": 2048,
+                    "coming_soon": true
                 },
                 {
                     "slug": "tinyllama-modal",
                     "model_type": "text2text",
-                    "context_window": 2048
+                    "context_window": 2048,
+                    "coming_soon": true
                 },
                 {
                     "slug": "mamba-modal",
                     "model_type": "text2text",
-                    "context_window": 128
+                    "context_window": 128,
+                    "coming_soon": true
                 },
                 {
                     "slug": "stable_lm2-modal",
                     "model_type": "text2text",
-                    "context_window": 2048
+                    "context_window": 2048,
+                    "coming_soon": true
                 },
                 {
                     "gemma": "gemma-modal",
                     "model_type":"text2text",
-                    "context_window": 2048
+                    "context_window": 2048,
+                    "coming_soon": true
                 },
                 {
                     "slug": "prem-1b-chat",
```

Comment:
Hi, I just came across the PR, however is it possible to not put in the json, otherwise number of keys are not same across all the blobs. Instead we can simply add:

```python
if model in ['prem-1b-chat', 'prem-1b-sum']:
    return "the logic on when coming soon ==  true"
```